### DISCUSSION TITLE
Be memory explosion when doing insert into select parallelly

### DISCUSSION THREAD

**Discussion by m3ngzhang** [apache/doris#13823] [A - General / Q&A]
I try to run like 10 exactly the same sql and use the same table at the same time, which is insert into select.
**What I expect:**
The memory usage would be 10 * single sql executing memory usage.
**What I found**
The memory usage y is much larger than what I thought, like 60GB vs 10GB.
So, I wanna ask how did this happen?

**Comment by kpfly** [comment]
@freemandealer @zhannngchen
